blueprints/controller-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('<%= friendlyTestDescription %>', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let controller = this.owner.factoryFor('controller:<%= controllerPathName %>').create();
```

Comment:
Can you tweak this to use `this.owner.lookup(‘controller:foo’);`? Controllers are intended to be singletons, so we don’t have to `factoryFor(...).create()`..